aws/onboarding/byok8s/python/rubrik_eks_cluster_private.py

```diff
@@ -0,0 +1,1208 @@
+"""EKS Cluster Setup."""
```

Comment:
This isn't too helpful. Please provide more context here.
Also, rename the script to `rubrik_create_private_eks_cluster.py`? And rename in the `README` also?